packages/http-server-javascript/src/scripts/scaffold/bin.mts

```diff
@@ -25,6 +25,14 @@ import { bifilter, indent } from "../../util/iter.js";
 import { createOnceQueue } from "../../util/once-queue.js";
 import { writeModuleFile } from "../../write.js";
 
+function tryGetOpenApi3(): Promise<typeof import("@typespec/openapi3") | undefined> {
+  try {
+    return import("@typespec/openapi3");
+  } catch {
+    return Promise.resolve(undefined);
+  }
```

Comment:
```suggestion
    return import("@typespec/openapi3").catch(() => undefined);
```
Since you're returning the `import` promise directly, the caller of this function would need to try/catch it. So either this can be made an `async` function and the `import` is awaited, or chain `catch` on import.

Reply:
Ah actually I did this in the one I wrote in `src/util` but forgot to just replace this function with a use of that one.

Reply:
Addressed in latest commits.